src/list-sorting/index.js

```diff
@@ -1,3 +1,35 @@
-function listSorting(needle, haystack) {}
+function listSorting(needle, haystack) {
+  let output = -1;
+
+  if(haystack.length < 1){
+    return output;
+  }
+
+  if(typeof haystack[0] !== "number"){
+    output = [];
+    let index = -1;
+    let arrayIndex;
+
+    for(let i=0; i<haystack.length; i++){
+      for(let j=0; j<haystack[i].length; j++){
+        if(haystack[i][j] === needle){
+          arrayIndex = i;
+          index = j;
+        }
+      }
+    }
+    output.push(arrayIndex, index);
+    return index !== -1 ? output : index;
+  }else{
```

Comment:
Missed an opportunity to use `Early Return` from last week. You could drop this `else`

Reply:
😮 It will be implemented going forward.